setup.cfg

```diff
@@ -65,19 +69,41 @@ all =
 
 
 [options.package_data]
-* = 
+* =
     peakdet/tests/data/*
 
 [flake8]
 exclude=
     *build/
-    heuristics
     tests
+    versioneer.py
 ignore = E126, E203, E402, W503
 max-line-length = 99
 per-file-ignores =
     */__init__.py:F401
 
+[isort]
+profile = black
+skip_gitignore = true
+extend_skip =
+    setup.py
+    versioneer.py
+    peakdet/_version.py
+skip_glob =
+    docs/*
+
+[pydocstyle]
+convention = numpy
+match =
+    peakdet/*.py
+match_dir = peakdet/[^tests]*
+
+[codespell]
+skip = venvs,.venv,versioneer.py,.git,build,./docs/_build
+write-changes =
+count =
+quiet-level = 3
+
```

Comment:
Might be worth adding black options as well?

Reply:
There is no support for `setup.cfg` [(issue)](https://github.com/psf/black/issues/683), we would have to add a `pyproject.toml` instead.
I don't see any desired configuration option other than perhaps the `--exclude` option.

However, I don't see any real benefit in excluding files like `versioneer.py` and not having style uniformity. Let me know your thoughts.